.github/workflows/ci-lint.yml

```diff
@@ -34,7 +34,7 @@ jobs:
         uses: actions/setup-python@v4.2.0
         with:
           python-version: '3.x'
-      - name: Set up Node.js 12
+      - name: Set up Node.js 18
```

Comment:
This just changes the name but not the actual version?

Reply:
ah ya